tests/unittests/quantizationTest.cpp

```diff
@@ -659,6 +659,78 @@ TEST(Quantization, quantizeSoftmaxAndLRN) {
   EXPECT_TRUE(qSM->getInput().getType()->isQuantizedType());
 }
 
+/// Test option to disable quantization of specific node kinds in the graph.
+TEST(Quantization, quantizeGraphPartially) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  auto *LHS = mod.createVariable(ElemKind::FloatTy, {3, 3}, "lhs",
+                                 VisibilityKind::Private, true);
+  auto *RHS = mod.createVariable(ElemKind::FloatTy, {3, 3}, "rhs",
+                                 VisibilityKind::Private, true);
+  auto *result = mod.createVariable(ElemKind::FloatTy, {3, 3}, "result");
+  LHS->getPayload().init(Tensor::InitKind::Xavier, 3, mod.getPRNG());
+  RHS->getPayload().init(Tensor::InitKind::Xavier, 3, mod.getPRNG());
+
+  auto *MMN = F->createMatMul("matmul", LHS, RHS);
+  auto *TN = F->createTanh("tanh", MMN);
+  F->createSave("ret", TN, result);
+
+  std::vector<NodeQuantizationInfo> QI{
+      {NodeQuantizationInfo::generateNodeOutputName(LHS->getName()), {0.3f, 0}},
+      {NodeQuantizationInfo::generateNodeOutputName(RHS->getName()), {0.4f, 0}},
+      {NodeQuantizationInfo::generateNodeOutputName(MMN->getName()), {0.6f, 0}},
+      {NodeQuantizationInfo::generateNodeOutputName(TN->getName()), {0.5f, 0}},
+  };
+
+  // Do not quantize the tanh.
+  KindSet doNotQuantize;
```

Comment:
does this not work: KindSet doNotQuantize = {Kinded::Kind::TanhNodeKind} ?

Reply:
Unfortunately not 😞